src/apiClient.js

```diff
@@ -2031,6 +2031,68 @@ class ApiClient {
         });
     }
 
+    uploadItemSubtitle(itemId, language, isForced, file) {
+        if (!itemId) {
+            throw new Error('null itemId');
+        }
+
+        if (!language) {
+            throw new Error('null language');
+        }
+
+        if (typeof isForced !== 'boolean') {
+            throw new Error('Invalid isForced value.');
```

Comment:
```suggestion
            throw new TypeError('parameter isForced must be a boolean.');
```

Reply:
I am confused with the syntax for error messages: some of them start with a capital letter, some doesn't, some end with dot. How could I harmonize them?

Reply:
We unfortunately don't really have any for now... This is still very much Emby's API client with a bunch of fixes to make it compatible with jellyfin-web.

What's important here is more the type of error being returned. I'm all for making it as precise as possible, to help to handle errors in clients using this. In this case, you are checking for a specific type and returning this error if it's not a valid type, so it is a `TypeError`.

My change to the message is more to reflect *why* the failure happened: the passed parameter is not a boolean. It helps a lot with debugging these :)

As for the format, feel free to choose one. We'll likely do a pass to make them more unified at some point.